### ISSUE TITLE
aria-atomic doesn't work as expected in Safari

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#27]
In https://github.com/phetsims/john-travoltage/issues/210 we found that the aria-atomic attribute doesn't work as expected in some cases in Safari.  We should create a simple test example using the attribute and submit a bug report if it sin't working in that case.  Otherwise, we should understand why it isn't working in a PhET sim.
